kinbot/zmatrix.py

```diff
@@ -115,8 +114,6 @@ def make_zmat_from_cart(species, rotor, cart, mode):
     for i in range(natom):
         if i == a or i == b or i == c or i == d:
             continue
-        #print(species.chemid, i, j, len(atom), len(zmat_atom))
-        #print(atom[i])
```

Comment:
delete these

Reply:
my matrix.py file has no print lines, so it is odd that they show on here. let me know if you still see them on the next push.